src/main/java/mekanism/common/tile/TileEntityFactory.java

```diff
@@ -406,11 +405,8 @@ public void sortInventory() {
                 for (int j = i + 1; j < inputSlots.length; j++) {
                     int checkSlotID = inputSlots[j];
                     ItemStack checkStack = inventory.get(checkSlotID);
-                    if (Math.abs(count - checkStack.getCount()) < 2) {
-                        continue;
-                    }
-                    if (!stack.isEmpty() && !checkStack.isEmpty() && !ItemHandlerHelper
-                          .canItemStacksStack(stack, checkStack)) {
+                    if (Math.abs(count - checkStack.getCount()) < 2 ||
```

Comment:
any idea what the `Math.min` is for here? @pupnewfster

Reply:
If I remember correctly it is to make it not run the code if the two slots are only one item apart in amount because it can't make them even as it does not divide evenly.